Solution

AngelaStrechen opened the pull request:
[DEMO LINK](https://AngelaStrechen.github.io/layout_landing-page/).

Commit:
Solution

Commit:
change readme.md

brespect reviewed (changes requested):
I can`t reach your DEMO Link, try to redeploy before submit:
<img width="2560" height="1282" alt="{4CE170D7-B68F-4F52-BF88-D173EF79C246}" src="https://github.com/user-attachments/assets/bcc04fc5-df7f-4698-9d71-fc8521ccb2fd" />

Commit:
change footer image

etojeDenys reviewed (changes requested):
1. add a hover effect to the header elements
<img width="1092" height="82" alt="image" src="https://github.com/user-attachments/assets/1c06c19c-128c-4c7e-909f-1b0db8b53e63" />

2. add Hoever to these links according to the layout
<img width="259" height="273" alt="image" src="https://github.com/user-attachments/assets/0f97b98d-6261-4254-99b2-adc595de6453" />

3. use higher quality images for the desktop version

4. download image without the mark
<img width="696" height="339" alt="image" src="https://github.com/user-attachments/assets/24868cd8-6fc1-4076-b011-5afb05b858b1" />

5. remove base outline
<img width="426" height="71" alt="image" src="https://github.com/user-attachments/assets/34d3a9bc-faad-42ef-b71c-15120aa46db4" />

6. disable text area resizing

Commit:
High-quality desktop images
, Clean image (no watermark)
, Removed input outline
, Disabled textarea resize
, Added hover effects

vitalii-bondarenko-ua reviewed (approved):
Looks good.
Prevent resize textarea.
<img width="952" height="677" alt="Screenshot 2025-11-29 at 09 23 58" src="https://github.com/user-attachments/assets/30b28cbf-e7a4-4b66-8d2b-64b63f9acefb" />
You can make it during preparing project for portfolio.
